test/integration/js/user_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'test_helper'
+
+class UserTest < ActionDispatch::IntegrationTest
```

Comment:
there's a problem putting it all in the same file because the other integration test uses a different driver, so i placed the driver change in the test itself and not in the setup.